app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -23,7 +23,7 @@ const equalsUnsorted = (arr1, arr2) => arr1.length === arr2.length
   && arr2.every(arr2Item => arr1.includes(arr2Item))
   && arr1.every(arr1Item => arr2.includes(arr1Item));
 
-const filterByCapabilities = (filterArray, modelToFilter) => API.get(`/api/${modelToFilter}?expand=resources&attributes=id,name,capabilities`)
+const filterByCapabilities = (filterArray, modelToFilter, isMulti) => API.get(`/api/${modelToFilter}?expand=resources&attributes=id,name,capabilities`)
```

Comment:
Could you please fix the above function `equalsUnsorted` lint error as well
Also, `filterByCapabilities` line is too long.
```
// storage manager functions:
const equalsUnsorted = (arr1, arr2) => arr1.length === arr2.length
  && arr2.every((arr2Item) => arr1.includes(arr2Item))
  && arr1.every((arr1Item) => arr2.includes(arr1Item));

const filterByCapabilities = (filterArray, modelToFilter, isMulti) =>
  API.get(`/api/${modelToFilter}?expand=resources&attributes=id,name,capabilities`)
    .then(({ resources }) => {
      const valueArray = [];
      resources.forEach((resource) => {
        const capsToFilter = resource.capabilities.map(({ uuid }) => uuid);
        if (equalsUnsorted(filterArray, capsToFilter)) {
          valueArray.push(resource);
        }
      });
```

Reply:
I'm sorry, I'm having problems with ESLint in my IDE, what's the lint error in equalsUnsorted? 
I assume something about the indents..

Reply:
&& arr2.every(**(arr2Item)** => arr1.includes(arr2Item)) 
&& arr1.every(**(arr1Item)** => arr2.includes(arr1Item));

it's actually a minor change.. we need to use the **arr1Item** and **arr2Item** inside brackets

Reply:
oh yeah sorry I always forget about the brackets before the arrow